### PR TITLE
Droidcon2015

### PR DESCRIPTION
**Pull request by mohadian**

### CODE REVIEW

File: external/_posts/2015-10-29-droidcon-uk-2015.markdown [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+categories: conferences
+image: /images/identity/droidcon2015.jpg
+excerpt: "The Droidcon London was once again a success in 2015, lots of developers attended from the UK and other countries. This year, Sky was a sponsor, gave talks and sent developers to attend the conferences. Sky had a stand were the company was promoted and was attracting talents to join our ranks.
+At NowTV, some developers attended the first day and others attended the second day of the Droidcon, here is a summary of what happened during these two days."
+---
+<h2>Summary</h2>
+<div>
+    <p>Here is a post to share what we have learnt at the <a href="https://skillsmatter.com/conferences/6712-droidcon-2015" class="external-link" rel="nofollow">Droidcon London 2015</a> conference that took place on the 29th and 30th of October 2015.</p>
+    <p>The Droidcon London was once again a success in 2015, lots of developers attended from the UK and other countries. This year, Sky was a sponsor, gave talks and sent developers to attend the conferences. Sky had a stand were the company was promoted and was attracting talents to join our ranks.</p>
```
Comment (sboundy) [inline external/_posts/2015-10-29-droidcon-uk-2015.markdown]: Quick typo to fix here - "Sky had a stand were the company..."  should read "Sky had a stand where the company"

Also, "attracting talents to join" probably better as "attracting talent to join"

File: external/_posts/2015-10-29-droidcon-uk-2015.markdown [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+	
+	<p> One of the first talks given at Droidcon was by Sky’s very own Andrew Jack. Andrew talked about the various ways of integrating video streaming into an Android app, including those built into the platform as well as third-party alternatives. It also covered the different codecs which are supported, and touched on issues such as DRM and compatibility.</p>
+	<p>More info: <a href="https://skillsmatter.com/skillscasts/6671-build-your-own-video-streaming-app">https://skillsmatter.com/skillscasts/6671-build-your-own-video-streaming-app</a></p>
+
+	<h3 id="DroidconLondon2015-AndroidforJavaDevelopers–ChetHaase"><span>Workshop: User centered design –<span style="color: rgb(128,128,128);"> Leonie Brewin</span></span></h3>
+
+	<p> Leonie Brewin is a designer at Novoda, and in this workshop (which spanned two sessions) attendees were put into groups and tasked with creating a design for a property-finding app. Leonie walked through various techniques and best practices for creating designs that best serve the needs of users, and how that process can be optimised to get designers, developers and product owners working closely together to produce high-quality applications.</p>
```
Comment (sboundy) [inline external/_posts/2015-10-29-droidcon-uk-2015.markdown]: Quick styling fix here - maybe try a quick rewrite on this sentence "Leonie walked through various techniques and best practices for creating designs that best serve the needs of users, and how that process can be optimised to get designers, developers and product owners working closely together to produce high-quality applications"

to split it into two.

File: external/_posts/2015-10-29-droidcon-uk-2015.markdown [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+	<ul>
+		<li><a href="https://github.com/google/iosched" class="external-link" rel="nofollow">Google io app 2015 source code</a>: a good example of animations</li>
+		<li><a href="https://github.com/googlesamples/android-topeka" class="external-link" rel="nofollow">https://github.com/googlesamples/android-topeka</a></li>
+		<li>Showcase of material design: <a href="https://github.com/nickbutcher/plaid" class="external-link" rel="nofollow">https://github.com/nickbutcher/plaid</a></li>
+	</ul>
+
+	<h3 id="DroidconLondon2015-AdvancedDevelopmentwiththeKotlinLanguage–JakeWharton">Advanced Development with the Kotlin Language&nbsp;– <span style="color: rgb(128,128,128);">Jake Wharton</span></h3>
+
+	<p><span  ><img  alt="Advanced Development with the Kotlin Language" height="250" width="333" src="/images/identity/droidcon/IMG_20151030_144522.jpg" title="Advanced Development with the Kotlin Language"></span></p>
+	<p>In this talk, Jake Wharton gave an intro to the Kotlin language and showed how improves Android development.</p>
```
Comment (sboundy) [inline external/_posts/2015-10-29-droidcon-uk-2015.markdown]: A quick typo here - "In this talk, Jake Wharton gave an intro to the Kotlin language and showed how improves Android development."
should read -
"In this talk, Jake Wharton gave an intro to the Kotlin language and showed how <b>it</b> improves Android development.

**Comment by sboundy** [comment]
A few comments on typos - but once fixed, rest looks good to me!

File: external/_posts/2015-10-29-droidcon-uk-2015.markdown [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    <p><span>The second day opened with a keynote given by Chet Haase. During this talk, Chet talked about differences between Java applications running on computers and Java running on the Android mobile platform.</span></p>
+    <p><strong>Key information:</strong></p>
+    <ul>
+    	<li>Application must not expect that their application is the main function of the user's phone: think battery and performance.</span></li>
+    	<li>Android uses the Java language but it has its own runtime (!= jre)</li>
+    	<li>Difference between server and mobile. Servers use the java runtime and run on fast CPU with lots of RAM. Mobile apps run on the Dalvik/Art runtime and a slow CPU/small RAM. The way garbage collection is done is different. On mobile, garbage collection is batched for when the phone is idle if possible.</span></li><li><span>The more memory allocations your application does, the more GC you will have, the slower the experience will be: try to re-use objects instead of creating new objects.</li>
```
Comment (craggy2593) [inline external/_posts/2015-10-29-droidcon-uk-2015.markdown]: "uses the java runtime" - Java should be capitalised

File: external/_posts/2015-10-29-droidcon-uk-2015.markdown [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
...rld"></span></p><p>Hoi Lam presented in this talk recommendations from Google on hardware and software for the Internet Of Things. It seems that Hoi wasn't allowed to share much information about his projects, brillo/weave, and this was a disappointment.</p><p>The talk was mainly about Google advertising their hardware (chromecast, Android TV, <a href="https://www.android.com/intl/en_uk/auto/" class="external-link" rel="nofollow">Android Auto</a>, <a href="https://developers.google.com/beacons/?hl=en" class="external-link" rel="nofollow">Google beacon</a>) and software (Android, <a href="https://developers.google.com/weave/" class="external-link" rel="nofollow">Weave</a> for IOT communication and <a href="https://developers.google.com/brillo/?hl=en" class="external-link" rel="nofollow">Brillo</a> for IOT hardware). However no technical details were given about the projects.</p><p>A funny annecdote was how they went about deciding if a device had heat issues that wouldn't be acceptable for customers: the cheek test. Managers have to test the devices by feeling it on their cheek, if it burns, it is an unacceptable amount of heat and needs to be addressed before commercialisation.</p>
```
Comment (craggy2593) [inline external/_posts/2015-10-29-droidcon-uk-2015.markdown]: "Hoi Lam presented in this talk recommendations from Google"

Possibly could be reworded? Talk recommendations sounds confusing to me

**Comment by craggy2593** [comment]
Added a few comments myself, but also looks fine to me.

**Comment by sboundy** [comment]
Looks good to me.

**Pull request merged**